server/app/views/applicant/ApplicantQuestionFragment.html

```diff
@@ -0,0 +1,9 @@
+<div th:fragment="applicant-question (question, messages)">
+    <th:block th:switch="${question.getType().getLabel()}">
```

Comment:
Annoying thing here is that I've not been able to access static values (like an Enum value) in Thymeleaf because it can't find the relevant class, so for now I put in this string comparison that makes me sad :(

Reply:
yes I tried that (which doesn't work because we don't use spring boot). Also tried the @ syntax referenced here https://www.thymeleaf.org/doc/tutorials/3.0/usingthymeleaf.html#preprocessing but no luck. Not giving up yet, since it seems like there has to be a way to do it, but haven't managed it yet

Reply:
Also played around with this a bit with no luck :( I tried adding the question type enum class to the template and it showed if I print it out in `<pre th:text="${#vars}"></pre>`, but couldn't figure out how to use it. Super annoying